### PR TITLE
[83120] Enable GA skip for OpenVINO in Dev Cloud environment

### PR DESCRIPTION
**Pull request by artyomtugaryov**
### All Submissions:
* [x] Create a meaningful name of the PR with reference to JIRA. Example: *[43714] [NIGHTLY] Take correct tag for snyk scans*
* [x] JIRA tickets:
  * 83120 - Model analyzer fails with topology type guessing
* [x] Set assignee to the PR.
* [x] Add corresponding labels to the PR. 
* [x] Set appropriate milestone to the PR.
* [x] Added tests for new functionality (unit / e2e)
* [x] Followed code style guidelines:
  * Code covered with typings
  * No debug printing in production code
  * No commented blocks of code in production code

### Available commands in pull request comments

1. `/bump:client` - bump to latest **master** branch in **client** submodule
1. `/bump:client:releases/2021/4` - bump to latest **releases/2021/4** branch in **client** submodule
1. `/bump:client:yatarkan:feature/yt/feature-name` - bump to latest **feature/yt/feature-name** branch from **yatarkan** remote fork in **client** submodule
1. `/update` - merge base (target) branch into target branch and push changes

> **NOTE**:
> For more details, please visit the [DEVELOPER.md](development/docs/DEVELOPER.md)

### CODE REVIEW

File: wb/main/scripts/templates/disable_telemetry.sh.jinja [hunk 1]
```diff
@@ -0,0 +1 @@
+[[ -f ${HOME}/intel/isip ]] || echo 0 >> ${HOME}/intel/isip
```
Comment (yatarkan) [inline wb/main/scripts/templates/disable_telemetry.sh.jinja:1]: new line?

File: wb/main/scripts/templates/setup.sh.jinja [hunk 2]
```diff
@@ -42,6 +42,10 @@ echo "Checking the OS of the target."
 
 source "${CURRENT_SCRIPT_PATH}/is_supported_os.sh"
 
+{% if disable_telemetry == True -%}
```
Comment (yatarkan) [inline wb/main/scripts/templates/setup.sh.jinja:45]:
```suggestion
{% if disable_telemetry -%}
```

File: wb/main/scripts/job_scripts_generators/setup_script_generator.py [hunk 3]
```diff
@@ -15,19 +15,42 @@
  limitations under the License.
 """
 from config.constants import NO_SUDO_SETUP_MESSAGE
+from wb.main.enumerates import PipelineTypeEnum
 from wb.main.scripts.job_scripts_generators.script_generator import ScriptGenerator, ScriptGenerationContext
 
 
 class SetupScriptGenerationContext(ScriptGenerationContext):
     NO_SUDO_SETUP_MESSAGE: str
+    TELEMETRY_SPECIFIC: str
```
Comment (yatarkan) [inline wb/main/scripts/job_scripts_generators/setup_script_generator.py:24]: Where is it used?

Comment (artyomtugaryov) [reply wb/main/scripts/job_scripts_generators/setup_script_generator.py]: nowhere

**Review by yatarkan — APPROVED** [review]

**Pull request closed without merging**